plutus-core/testlib/PlutusIR/Generators/QuickCheck/ShrinkTerms.hs

```diff
@@ -82,7 +82,9 @@ fixupTerm_ :: TypeCtx
 fixupTerm_ tyctxOld ctxOld tyctxNew ctxNew tyNew tm0 =
```

Comment:
Note to self: `tyctxNew` and `ctxNew` aren't used, are they? Try simplifying this whole thing.